ffw/UIRPC.js

```diff
@@ -412,9 +412,18 @@ FFW.UI = FFW.RPCObserver.create(
               if(request.params.enableSeek) {
                 this.OnSeekMediaClockTimer(request.params.startTime, request.params.appID);
               }
+              
               if(request.params.audioStreamingIndicator) {
                 SDL.SDLController.SetAudioStreamingIndicator(request.params)
+              } 
+              else if (SDL.SDLController.model.mediaPlayerIndicator !== 
+                SDL.SDLModel.data.mediaPlayerIndicatorEnum.PLAY_PAUSE) {                
+                modifiedParams = Object.assign({}, request.params, {
+                  "audioStreamingIndicator": "PLAY_PAUSE"
+                });
+                SDL.SDLController.SetAudioStreamingIndicator(modifiedParams)
               }
```

Comment:
```suggestion
              
              var indicator = request.params.audioStreamingIndicator ? request.params.audioStreamingIndicator : "PLAY_PAUSE"
              SDL.SDLController.SetAudioStreamingIndicator(indicator)

```
Seems a bit odd to be sending in a modified version of the message. I would suggest editing the function to take in the indicator rather than the full message, as most of it is unused in the function at the moment